libs/SmartSync/src/com/salesforce/androidsdk/smartsync/util/SyncDownTarget.java

```diff
@@ -126,6 +127,15 @@ public JSONObject asJSON() throws JSONException {
     public abstract JSONArray continueFetch(SyncManager syncManager) throws IOException, JSONException;
 
     /**
+     * Fetches list of IDs still present on the server from the list of local IDs.
+     *
+     * @param syncManager SyncManager instance.
+     * @param localIds Local IDs from SmartStore.
+     * @return List of IDs still present on the server.
+     */
+    public abstract Set<String> getListOfRemoteIds(SyncManager syncManager, Set<String> localIds);
```

Comment:
Abstract method, so that custom sync down targets can choose how to deal with this.